ui-tests/src/test/java/org/openmrs/reference/FormTest.java

```diff
@@ -0,0 +1,108 @@
+package org.openmrs.reference;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.openmrs.reference.groups.BuildTests;
+import org.openmrs.reference.page.AdministrationPage;
+import org.openmrs.reference.page.ClinicianFacingPatientDashboardPage;
+import org.openmrs.reference.page.HeaderPage;
+import org.openmrs.reference.page.HtmlFormsPage;
+import org.openmrs.reference.page.ManageHtmlFormsPage;
+import org.openmrs.reference.page.HomePage;
+import org.openmrs.reference.page.ManageFormsPage;
+import org.openqa.selenium.By;
+import org.openmrs.uitestframework.test.TestBase;
+
+import static org.junit.Assert.assertNotNull;
+
+/**
+ * Created by nata on 24.06.15.
+ */
+public class FormTest extends TestBase {
+
+	private static String name = "newFormTest1";
+	private static String description = "description of new form";
+	private static String version = "1.2";
+	private HomePage homePage;
+	private AdministrationPage administrationPage;
+	private HeaderPage headerPage;
+	private ManageFormsPage manageForm;
+	private ManageHtmlFormsPage manageHtmlFormsPage;
+	private HtmlFormsPage htmlFormsPage;
+	private ClinicianFacingPatientDashboardPage patientDashboardPage;
+
+	@Before
+	public void setUp() throws Exception {
+		homePage = new HomePage(page);
+		assertPage(homePage.waitForPage());
+		headerPage = new HeaderPage(driver);
+		administrationPage = new AdministrationPage(page);
+		manageForm = new ManageFormsPage(driver);
+		htmlFormsPage = new HtmlFormsPage(page);
+		manageHtmlFormsPage = new ManageHtmlFormsPage(page);
+		patientDashboardPage = new ClinicianFacingPatientDashboardPage(page);
+	}
```

Comment:
You need to indent `setUp() `method to match with the existing ui test indention

Reply:
Where exactly need to  be indented, these identation is the same as whole commit